hack/RELEASE.md

```diff
@@ -0,0 +1,23 @@
+# Libpod/Podman/Podman Remote Release Checklist
+
+0. Make a new branch using the name format <1.2.3>[-<epic>]
```

Comment:
Usually we do not include micro - just Major.Minor - in branch names (branches will include multiple micro releases)

Reply:
Sure, but automation is really naive and prefers to be told explicitly what to do :grin:

I see some branches named like '1.2.3-rc4'.  Managing those should also be included under the release-automation purview IMHO.  

So something like [this regex-format  sounds like it would be more accurate:
``^[A-z\-]+ Release [0-9]+\.[0-9]+(\.[0-9]+(-[\w\-\.]+)?)?``](https://regex101.com/r/Jr68bt/1)

?

Reply:
Are you sure those aren't tags?

Release branches (aside from one unfortunate case we will never, ever repeat) have always been `vX.Y`

Reply:
That regex is just matching the commit comment, so automation knows to start either a dummy release (for a PR) or a real one (for a branch).  I'm happy to make the regex whatever you prefer (hence the playground link above).  So you'd [prefer `^[A-z\-]+ Release (v[0-9]+\.[0-9]+(\.[0-9]+(-[\w\-\.]+)?)?)`](https://regex101.com/r/Jr68bt/2)

Reply:
> aside from one unfortunate case we will never, ever repeat

This is a good sign that automation is needed.  It prefers to ***always*** repeat it self :grin:

Reply:
We'll need special handling for release candidates (`vA.B.C-rcD`) that explicitly marks them as pre-release in Github, so we may need two regexes?

Reply:
That's fine, the `(-[\w\-\.]+)?)?)` at the end should allow for that.  Practically speaking, what is the difference with a pre-release.  For example, are there steps that should not be performed?

Reply:
The biggest thing is that pre-releases must mark the Github release as "pre-release", and do not need to build Windows/OS X/etc binaries (but it's not necessarily bad if it does build them).

Reply:
Okay, so automation should make a github release, named using the form 'vX.Y' _UNLESS_ it's a pre-release as indicated by finding the string 'rc' inside the match-group `(-[\w\-\.]+)?)?)`.  When it's a pre-release, then the github-release MUST include the entire tag name (so 'vX.Y.Z-rcBlAH123WHatEvER`).  Correct?